commands/create_command.py

```diff
@@ -1,35 +1,43 @@
-from commands.Icommand import ICommand
+import json
+import os
+from typing import TextIO
+
+from Icommand import ICommand
+from schema_keys import SchemaKeys
 from response.exceptions import *
-import os, json
-from commands.schema_keys import SchemaKeys
 
 parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
 
+
 class CreateCommand(ICommand):
     def __init__(self, schema_path):
         self.schema_path = schema_path
-        self.__validate_and_load_schema()  
+        self.__validate_and_load_schema()
 
     def __validate_and_load_schema(self):
-        if (self.schema_path == None or not os.path.exists(os.path.join(parent_dir, self.schema_path))):
+        if self.schema_path is None or not os.path.exists(os.path.join(parent_dir, self.schema_path)):
             raise FileNotFound("Schema path not found")
-        try:  
-            with open (self.schema_path, "r") as schema_file:
+        try:
+            with open(self.schema_path, "r") as schema_file:
                 self.data = json.load(schema_file)
         except:
             raise ErrorLoadingFile("Error loading Json file")
 
         if not SchemaKeys.DATABASE in self.data:
             raise MissingDataError("Database name is missing")
-        
-    def execute(self):  
+
+    def execute(self):
         self.__create_database()
         self.__create_tables()
+        for table in self.data[SchemaKeys.TABLES]:
+            t_path = os.path.join(self.path, table[SchemaKeys.NAME])
+            self.create_table_schema(t_path, table)
+            self.create_indices(t_path, table)
```

Comment:
I recommend to habe a `TableMetadata` class. This class takes in consructor table schema. 
Also I would suggest to have Index class. 

```
class TableMetadata
     List<Index> indeces;
      name;
      primaryKey

    serialize() {
        json.write()
        for (index in indices )
             index.serialize()
    }
    

```